### ISSUE TITLE
why the logits are different in conformer and deit

### ISSUE THREAD

**Issue by shapera-lab** [vasgaowei/TS-CAM#20]
conformer logits
https://github.com/vasgaowei/TS-CAM/blob/master/lib/models/conformer.py#L460
